conf/src/main/java/org/apache/nemo/conf/JobConf.java

```diff
@@ -332,6 +316,22 @@
   public final class ExecutorId implements Name<String> {
   }
 
+  /**
+   * Maximum off-heap memory ratio to the total memory in the executor.
+   */
+  @NamedParameter(doc = "The maximum ratio of off-heap memory size to the total memory size.",
+    short_name = "max_offheap_ratio", default_value = "0.02")
```

Comment:
Why `0.02`?  `0.02` is smaller than the previous default value `0.2`, but it is basically same in that some portion of executor on-heap memory is allocated in an unnoticeable way. I think we'd better make it to `0`, and enable off-heap memory management only when this parameter becomes bigger than 0.

Reply:
changed default value to 0.2, please check.